ql/src/test/results/clientpositive/llap/probedecode_mapjoin_stats.q.out

```diff
@@ -416,4 +416,4 @@ POSTHOOK: Input: default@orders_fact
 POSTHOOK: Input: default@seller_dim
 #### A masked pattern was here ####
 101	101	12345	12345	Seller 1	Item 101	2001-01-30 00:00:00
-104	104	23456	23456	Seller 2	Item 104	2002-03-02 00:00:00
+104	104	23456	23456	Seller 2	Item 104	NULL
```

Comment:
Why do we get this NULL value?

Reply:
Because of the following query in probedecode_mapjoin_stats.q

INSERT INTO orders_fact values(23456, 104, '2002-02-30 00:00:00'); 

Also timestamp format is "YYYY-MM-DD HH:MM:SS" due to that '2002-02-30 00:00:00' value get converted to null. Since probedecode_mapjoin_stats.q is to test happy flow of mapjoin_stats so lets make it some valid date to avoid confusion.